src/components/inline_edit/inline_edit_form.tsx

```diff
@@ -171,6 +167,14 @@ export const EuiInlineEditForm: FunctionComponent<EuiInlineEditFormProps> = ({
   const [isEditing, setIsEditing] = useState(false || startWithEditOpen);
   const [editModeValue, setEditModeValue] = useState(defaultValue);
   const [readModeValue, setReadModeValue] = useState(defaultValue);
+  const showPlaceholder = placeholder && !readModeValue;
+
+  const readModeStyles = euiInlineEditReadModeStyles(euiTheme);
+  const readModeCssStyles = [
+    readModeStyles.euiInlineEditReadMode,
+    isReadOnly && readModeStyles.isReadOnly,
+    showPlaceholder && readModeStyles.hasPlaceholder,
+  ];
```

Comment:
Very minor cleanup - since we're only using `showPlaceholder` in one place and for the actual rendering logic, let's go ahead and remove the var abstraction - it doesn't feel necessary 

```suggestion

  const readModeStyles = euiInlineEditReadModeStyles(euiTheme);
  const readModeCssStyles = [
    readModeStyles.euiInlineEditReadMode,
    isReadOnly && readModeStyles.isReadOnly,
    placeholder && !readModeValue && readModeStyles.hasPlaceholder,
  ];
```